org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -664,6 +664,48 @@ SsoImageCentralAmiLibrarian:
       - 'arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore'
     sessionDuration: 'PT12H'
 
+SsoKubernetesDeveloper:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.3.2/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-kubernetes-developer'
+  StackDescription: 'SSO: Limited developer role used by kuberenetes developers'
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account:
+        - !Ref SandboxAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref TBD
+    permissionSetName: 'Kubernetes-Developer'
+    sessionDuration: 'PT12H'
+    inlinePolicy: >-
```

Comment:
I don't think this policy, by itself, is enough for users to use or manage EKS.  It looks like there are a lot of different features of EKS and policy you apply depends on what EKS features you setup.  I would suggest you combine the `arn:aws:iam::aws:policy/PowerUserAccess` policy with a few of the EKS inline or managed policies.  Again it depends on which EKS options you setup.  I would take a look here for an explanation of inline policies that are required, https://docs.aws.amazon.com/eks/latest/userguide/security_iam_id-based-policy-examples.html   and take a look here for the managed policies, https://docs.aws.amazon.com/eks/latest/userguide/security-iam-awsmanpol.html

You could probably test those out manually in sandbox before deciding which set to go with?

Reply:
Thanks Khai.  I have actually looked at those and will have to test some of those policies out.  I think the most confusing part is that all of these policies seem to allow people to create, list and manage the cluster, but theres not really something that is obvious to me that allows for people to just "use the cluster".   

For instance, I don't know whether a user would need permission to "ec2:createInstance" since sometimes when you deploy a kubernetes deployment, it may spin up a new instance.